internal/cmd/byok/command_describe.go

```diff
@@ -0,0 +1,97 @@
+package byok
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	errorMsg "github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type describeStruct struct {
+	Id        string   `human:"Id" serialized:"id"`
+	Key       string   `human:"Key" serialized:"key"`
+	Roles     []string `human:"Roles" serialized:"roles"`
+	Provider  string   `human:"Provider" serialized:"provider"`
+	State     string   `human:"State" serialized:"state"`
+	CreatedAt string   `human:"Created At" serialized:"created_at"`
+	UpdatedAt string   `human:"Updated At" serialized:"updated_at"`
+	DeletedAt string   `human:"Deleted At" serialized:"deleted_at"`
+}
+
+func (c *command) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe <id>",
+		Short:             "Describe a self-managed key in Confluent Cloud.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE:              c.describe,
+	}
+
+	cmd.Flags().Bool("show-policy-command", false, "Print post-creation step to grant Confluent access to the key.")
```

Comment:
We try to keep flags as short as possible. This one's a bit of a mouthful. How about `--policy-command` or `--show-policy`?

Reply:
Fixed.